statuspage/src/components/CanaryModal.vue

```diff
@@ -0,0 +1,60 @@
+<!-- This component encapsulates a modal that is displayed -->
+<!-- when clicking on canary                               -->
+<template>
+  <b-modal :id="'modal-canary'+name+namespace" ok-only>
+    <template #modal-title>
+      <img :src="'images/' + checkType + '.svg'" height="30px"> {{name}}
```

Comment:
Should use template literals. Then, `rem` values over `px` values are preferred as `rem` can be scaled against base font size. See: https://nekocalc.com/px-to-rem-converter

Change the base font size to see what I mean.

At some stage, if `rem`s are consistently used, we can use the base font size to scale everything, meaning we'd need fewer media queries.

Reply:
Changed :)